src/init.php

```diff
@@ -421,13 +421,6 @@ function alps_gutenberg_blocks_render_block_latest_post($attributes) {
  * Registers the `core/latest-posts` block on server.
  */
 function register_block_alps_latest_posts() {
-	wp_enqueue_script(
-		'gutenberg-blocks-block-js', // Handle.
-		plugins_url('/dist/blocks.build.js', dirname(__FILE__)), // Block.build.js: We register the block here. Built with Webpack.
-		array('wp-blocks', 'wp-i18n', 'wp-element', 'wp-editor', 'wp-data'), // Dependencies, defined above.
-		// filemtime(plugin_dir_path(__DIR__) . 'dist/blocks.build.js'), // Version: filemtime — Gets file modification time.
-		true // Enqueue the script in the footer.
-	);
```

Comment:
This this function out, the edit pages crash. 

![2020-07-27 at 6 21 PM](https://user-images.githubusercontent.com/132864/88597826-1fe2f380-d036-11ea-9c96-0be90be0e2fe.png)